pyk/.gitignore

```diff
@@ -1,8 +1,4 @@
-/dist/
```

Comment:
If there's a way, it'd be nice to keep this modular:
* `.gitignore` does not apply in `pyk`
* `pyk/.gitignore` applies in `pyk`